dvga/routes.py

```diff
@@ -0,0 +1,189 @@
+import jwt 
+import graphene 
+
+from datetime import datetime
+from datetime import timedelta
+from flask import Flask
+from flask import request
+from flask import jsonify
+from flask import render_template
+from flask import redirect
+from flask import make_response
+from flask import session
+from flask_graphql import GraphQLView
+
+import config
+from app import app
+from app import db
+from stuffing import solutions as list_of_solutions
+from .decorators import token_required
+from .decorators import access_controlled_for
```

Comment:
Clever, I like the access controlled for middleware and I like how the authentication and authorization work here.

Did we include any write up in README to indicate the new scenarios?

Reply:
Nope. As I have written in the PR comments thread, I did not write the solutions yet. I you also think a paragraph in the README is needed, I'll write it.